package-versions/grouping-and-overrides/transitives.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: default
+title: Transitives in Groups
+parent: Grouping and Overrides
+grand_parent: Package Versions
+nav_order: 3
+---
+
+# Transitive Dependency Handling in Groups
+
+Managing transitive dependencies within groups is similar to [how NuGet resolves package dependencies for `<PackageReference>`](https://docs.microsoft.com/en-us/nuget/concepts/dependency-resolution).
+
+## Global Group
+
+When no `CentralManagementGroup` is defined, the `<PackageVersion>` from the global group is applied.
+
+![global group transitive handling](/assets/images/global-group.png){: .transitive-diagram }
+
+In this example, a package imported from the global group will import a transitive dependency from the global group.
+
+## Single Group
+
+When a direct dependency imports a transitive dependency that is also defined within the same group, that version will be used.
+
+![single group transitive handling](/assets/images/single-group.png){: .transitive-diagram }
+
+In this example, a direct dependency within `CentralManagementGroup="A"` imports a transitive dependency that is also found within `CentralManagementGroup="A"`, so that version is used.
+
+## Single Group with Global Fallback
+
+If no `<PackageVersion>` can be found for a package within the same `CentralManagementGroup`, the transitive dependency will fallback to the global group.
+
+![single group with global fallback transitive handling](/assets/images/single-group-with-global-fallback.png){: .transitive-diagram }
+
+In this example, there is no package version available within the same group as the direct dependency; however, a package version has been specified within the global group for this package. In this case, the project will import the transitive dependency with the version found in the global group.
+
+## Multiple Groups with a Single Group Defined
+
+When more than one `CentralManagementGroup` imports the same package as a transitive dependency but only one `CentralManagementGroup` has a `<PackageVersion>` defined for the transitive package, that version will be used.
+
+![global group transitive handling](/assets/images/multiple-groups-with-single-group-defined.png){: .transitive-diagram }
+
+In this example, two different packages in two different `CentralManagementGroup` import the same transitive dependency. Because `CentralManagementGroup="A"` has a `<PackageVersion>` for the transitive dependency but `CentralManagementGroup="B"` does not, the version for `CentralManagementGroup="A"` will be used.
```

Comment:
The image sounds like somethings I don't expect to see happening at all.
If groups are here to isolated dependency tree, I was love to see resolution fails by default rather than picking it from another group

This will lead to headake when trying to understand why downgrade can happen.

If you still think this could be beneficial to someone, I'd rather hint to the user that this could not be resolved because etc ...

Group should probably be fully isolated by default, but if you want to to ignore this I would love an OPT_IN "yes you can go pick in global/or this group/"
Just like the Scope of the `PackageSource`